proto/imuachain/oracle/v1/index_recent_params.proto

```
@@ -1,7 +1,7 @@
 syntax = "proto3";
-package exocore.oracle.v1;
+package imuachain.oracle.v1;
```

Buf (1.47.2)

Files with package "imuachain.oracle.v1" must be within a directory "imuachain/oracle/v1" relative to root but were in directory "proto/imuachain/oracle/v1".

(PACKAGE_DIRECTORY_MATCH)